pyfr/plugins/fluidforce.py

```diff
@@ -99,9 +111,15 @@ def __init__(self, intg, cfgsect, suffix):
 
                 eidxs[etype, fidx].append(eidx)
                 norms[etype, fidx].append(mpn[:, None]*npn)
+                # Get the flux points position of the given face and element
+                # indices relative to the moment origin
+                if self._mcomp:
+                    rfpt = eles.get_fpts(eidx, fidx) - morigin
+                    rfpts[etype, fidx].append(rfpt)
```

Comment:
Can we bake in the cross product with the normals here?

Reply:
I thought about that. This is perfectly fine for the pressure forces, since the pressure is a scalar. However, for the viscous forces we have `\vector{\rfpt} \cross ( \tensor{\tau} \cdot \vector{\normal})`. I'll have to simplify the latter with Einstein notation to see if the term `\vector{\rfpt} \cross \vector{\normal}` appears in it.

Reply:
If there are no tricks we can play down the line, it might be easier to not bake in the cross products.

Reply:
The comment suggests these points are relative, but they look absolute to me?

Reply:
My bad, I lost `morigin`  at some point. I fixed it